docx/oxml/text/symbol.py

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+
+"""
+Custom element classes related to text symbols (CT_Sym).
+"""
+
+from ..xmlchemy import (
+    BaseOxmlElement, OptionalAttribute
+)
+from ..simpletypes import (
+    ST_String
+)
+from warnings import warn
+
+# this font face uses WordPerfect, it maps usual chars to special ones
+map_WP_TypographicSymbols = {
+    '0027': "§",
+    '0040': "”",
+    '0038': "©",
+    '003D': "’",
+    '0041': "“",
+    '0042': "–",
+    '0043': "—"
+}
+
+class CT_Sym(BaseOxmlElement):
+    """
+    ``<w:sym>`` element, containing the symbol.
+    """
+    char = OptionalAttribute('w:char', ST_String)
+    font = OptionalAttribute('w:font', ST_String)
+
+    @property
+    def readSymbol(self):
+        """
+        Returns the symbol value if the font is supported otherwise returns the Unicode value with an warning.

```

Comment:
Small typo: `Returns the symbol value if the font is supported otherwise returns the Unicode value with a warning.`